frontend/src/config/Config.ts

```diff
@@ -130,7 +130,11 @@ const conf = {
       istioConfig: (namespace: string) => `api/namespaces/${namespace}/istio`,
       allIstioConfigs: () => `api/istio/config`,
       istioConfigCreate: (namespace: string, objectType: string) => `api/namespaces/${namespace}/istio/${objectType}`,
-      istioConfigDetail: (namespace: string, objectType: string, object: string) =>
+      istioConfigDetail: (cluster: string, namespace: string, objectType: string, object: string) =>
+        `api/namespaces/${namespace}/istio/${objectType}/${object}?cluster=${cluster}`,
```

Comment:
I don't think query params belong here. These are just about constructing the required path. Query params are treated differently since they're optional and adding them here could cause issues with other things trying to construct the URL with its own query params.